<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medicalajout</class>
 <widget class="QDialog" name="medicalajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Forlmulaire Ajouter Stock </string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reference</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeeM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Marque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantité</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="referenceM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="marqueM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantiteM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantité reservé</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="ajoutermMPB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reserveM">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IDML">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID_Medical</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDM">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="stockmedicalPB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Stock Medical</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="stocktechniquePB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock Technique</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TailleL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Taille</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TailleLE">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="couleurLE">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="couleurL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>couleur</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDFL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID_technique</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDF">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="delaiduL">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delai_du</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="delaiduDE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="delaiauL">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delai_au</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="delaiauDE">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
